### PR TITLE
5390: [TEST]: Stabilize flaky Throttling Reroute test

### PR DESCRIPTION
**Pull request by izadorozhna**
Implements #5390

* Fixed the verification inside the test to verify the precise time difference between reroute request and action reroute start

### CODE REVIEW

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ThrottlingRerouteSpec.groovy [hunk 1]
```diff
@@ -207,9 +219,18 @@ class ThrottlingRerouteSpec extends HealthCheckSpecification {
...
-            assert northbound.getLink(brokenIsl).state == IslChangeType.FAILED
+        if (withWait) {
+            Wrappers.wait(WAIT_OFFSET, 0) {
+                assert northbound.getLink(brokenIsl).state == IslChangeType.FAILED
+            }
         }
         return brokenIsl
     }
+
+    Long convertStringTimestampIsoToLong(String timestampIso) {
```
Comment (pkazlenka) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ThrottlingRerouteSpec.groovy:230]: I'd keep this as a static method in an Object/Interface like DateTimeConverter.

**Review by pkazlenka — APPROVED** [review]

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ThrottlingRerouteSpec.groovy (hunk 1 above)
Comment (izadorozhna) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ThrottlingRerouteSpec.groovy]: Done, made the method static, and moved to the FlowHelper. Currently we are trying to get rid of such helpers however I could not find where I can put such static method (technically it is related to the flow timestamp). However, if you prefer, I can create a separate class such as DateTimeConverter,

Comment (pkazlenka) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ThrottlingRerouteSpec.groovy]: > Done, made the method static, and moved to the FlowHelper. Currently we are trying to get rid of such helpers however I could not find where I can put such static method (technically it is related to the flow timestamp). However, if you prefer, I can create a separate class such as DateTimeConverter,

FlowHelper is ok, anyway we're planning to replace it with FlowExtended.

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ThrottlingRerouteSpec.groovy [hunk 2]
```diff
@@ -198,7 +208,7 @@ class ThrottlingRerouteSpec extends HealthCheckSpecification {
      * @param flowpath path to break
      * @return ISL which 'src' was brought down in order to break the path
      */
-    Isl breakFlow(FlowPathPayload flowpath) {
+    Isl breakFlow(FlowPathPayload flowpath, boolean withWait = true) {
```
Comment (yuliiamir) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ThrottlingRerouteSpec.groovy:211]: minor: withWait  -> waitForBrokenIsl

Comment (izadorozhna) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ThrottlingRerouteSpec.groovy]: done, renamed

**Review by yuliiamir — APPROVED** [review]

**Pull request merged**